commit ce90d8fb9beabe991e27740c464f95ac86d74b20
Author: Tatsuya Kinoshita <tats@debian.org>
Date:   Mon Mar 15 19:07:39 2021 +0900

    New patch fix-empty-content.patch to display email body (Closes: #981066)

diff --git a/debian/patches/fix-empty-content.patch b/debian/patches/fix-empty-content.patch
new file mode 100644
index 0000000..65412d6
--- /dev/null
+++ b/debian/patches/fix-empty-content.patch
@@ -0,0 +1,29 @@
+Subject: Use stdout as controlling terminal
+Origin: backport, https://git.sr.ht/~sircmpwn/aerc/commit/dc281e46d2aceaab6a7b7a290f9af89fef46159d
+Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=983165
+Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=981066
+
+    Soves an issue with go1.15 not letting ctty be a parent. See
+    https://github.com/creack/pty/pull/97 for more details.
+
+diff --git a/widgets/terminal.go b/widgets/terminal.go
+index 8fc38ce..77da71e 100644
+--- a/widgets/terminal.go
++++ b/widgets/terminal.go
+@@ -4,6 +4,7 @@ import (
+ 	"os"
+ 	"os/exec"
+ 	"sync"
++	"syscall"
+ 
+ 	"git.sr.ht/~sircmpwn/aerc/lib/ui"
+ 
+@@ -237,7 +238,7 @@ func (term *Terminal) Draw(ctx *ui.Context) {
+ 
+ 		if term.pty == nil {
+ 			term.vterm.SetSize(ctx.Height(), ctx.Width())
+-			tty, err := pty.StartWithSize(term.cmd, &winsize)
++			tty, err := pty.StartWithAttrs(term.cmd, &winsize, &syscall.SysProcAttr{Setsid: true, Setctty: true, Ctty: 1})
+ 			term.pty = tty
+ 			if err != nil {
+ 				term.Close(err)
diff --git a/debian/patches/series b/debian/patches/series
index 3b5d2bb..6e8232a 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -2,3 +2,4 @@ use-creack-pty.diff
 fix-go-maildir-usage.diff
 temp-disable-beep.diff
 fix-installation-directories.diff
+fix-empty-content.patch
